pages/linux/pkg-config.md

```diff
@@ -0,0 +1,11 @@
+# pkg-config
+
+> Return metainformation about installed libraries.
```

Comment:
```suggestion
> Tool to provide the details of installed libraries for compiling applications
```

Reply:
@andrik Maybe this is even better?
```suggestion
> Provide the details of installed libraries for compiling applications.
```

Reply:
@zdroid I updated your suggestion adding `More information` section